### PR TITLE
Introduce policy for new images/packages

### PR DESCRIPTION
**Pull request by mathbunnyru**
## Describe your changes

I'm combing @consideRatio and @minrk ideas from https://github.com/jupyter/docker-stacks/issues/1958 and my proposal of voting from https://github.com/jupyter/docker-stacks/pull/2000.

## Issue ticket if applicable

Fix: https://github.com/jupyter/docker-stacks/issues/1958

## Checklist (especially for first-time contributors)

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] I will try not to use force-push to make the review process easier for reviewers
- [ ] I have updated the documentation for significant changes

### CODE REVIEW

File: docs/maintaining/new-images-and-packages-policy.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+   - Does the package open additional ports, or add new web endpoints, that could be exploited?
+
+With all this in mind, we have a voting group, which consists of
+[mathbunnyru](http://github.com/mathbunnyru),
+[consideRatio](http://github.com/consideRatio),
+[yuvipanda](https://github.com/yuvipanda) and
+[minrk](https://github.com/minrk).
+
+This voting group is responsible for accepting or declining new packages and stacks.
+The change is accepted, if there are **at least 2 positive votes**.
```
Comment (mathbunnyru) [inline docs/maintaining/new-images-and-packages-policy.md:35]: @consideRatio @yuvipanda @minrk how do you feel about this?

**Review by consideRatio — APPROVED** [review]
I think this is a great starting point if not, and I don't come up with suggested changed to work before merge. I think this should be approved by more people than me though!

**Comment by yuvipanda** [comment]
Honored to be a part of this, and looks good to me as a great start. I would also nominate @manics to be a part of this group if you feel ok with it, given he does a lot of work on repo2docker as well.

**Comment by minrk** [comment]
I think the principle makes good sense. I don't feel like I'm contributing much here, so I think I'd prefer not to be on the voting list.

**Comment by mathbunnyru** [comment]
> Honored to be a part of this, and looks good to me as a great start. I would also nominate @manics to be a part of this group if you feel ok with it, given he does a lot of work on repo2docker as well.

Thanks for the suggestion! @manics I added you to the list. Please, tell me if you feel comfortable with this.

**Comment by mathbunnyru** [comment]
> I think the principle makes good sense. I don't feel like I'm contributing much here, so I think I'd prefer not to be on the voting list.

Ok, I removed you from the list. If you ever change your mind, please, let me know 🙂

**Comment by manics** [comment]
> Thanks for the suggestion! @manics I added you to the list. Please, tell me if you feel comfortable with this.

Yes, I'm happy to help, thanks for asking!

File: docs/maintaining/new-images-and-packages-policy.md (hunk 1 above)
Comment (consideRatio) [reply docs/maintaining/new-images-and-packages-policy.md]: :+1: for the simplicity of this!

**Pull request merged**